lib/HeroBonus.cpp

```diff
@@ -2445,7 +2433,7 @@ JsonNode CreatureAlignmentLimiter::toJsonNode() const
 	JsonNode root(JsonNode::JsonType::DATA_STRUCT);
 
 	root["type"].String() = "CREATURE_ALIGNMENT_LIMITER";
-	root["parameters"].Vector().push_back(JsonUtils::stringNode(EAlignment::names[alignment]));
+	root["parameters"].Vector().push_back(JsonUtils::stringNode(GameConstants::ALIGNMENT_NAMES[static_cast<std::underlying_type<EAlignment>::type>(alignment)]));

```

Comment:
I get that you want to do it the right way(tm). But readability...

Reply:
It is a draft now, will be split by multiple PRs (native terrain PR will be first, then I will finish teleport).

Reply:
moved to #1876